CITATION.cff

```diff
@@ -2,8 +2,8 @@ cff-version: 1.2.0
 message: If you use this software, please cite it using the metadata from this file.
 type: software
 title: 'mrremind: MadRat REMIND Input Data Package'
-version: 0.221.1
-date-released: '2025-03-13'
+version: 0.221.1.9001
```

Comment:
Dev versions are useful for testing inputdata generation on the cluster. Don't forget to run buildLibrary again before merging and incrementing either second or third place number.